src/java.base/share/classes/java/net/doc-files/net-properties.html

```diff
@@ -267,13 +267,21 @@ <H2>Address Cache</H2>
 	policy, while a value of 0 (zero) means no caching. The default value
 	is -1 (forever) if a security manager is installed, and implementation-specific
 	when no security manager is installed.</P>
+	<LI><P><B>{@systemProperty networkaddress.cache.stale.ttl}</B> (default: see below)<BR>
+	Value is an integer corresponding to the number of seconds the stale names
+	will be kept in the cache. The name is considered stale if it expired and
+	attempts to refresh it were unsuccessful. This property is useful if it is
+	preferable to use a stale name rather than result of unsuccessful name lookup.
+	A value of 0 (zero) or if the property is not set means do not use stale
+	names, the negative value is ignored.
+	The default value is implementation-specific.</P>
```

Comment:
Maybe we should say that the default value is 0 since this is what is documented in `java.security`

Reply:
In the "java.security" file we documented the default value of the property in that file but it is commented out. Right now the new property specified in the same way as the old "networkaddress.cache.ttl", see [net-properties.html](https://github.com/openjdk/jdk/pull/13285/files/ce595f74f511553274c9da254d53b368abe44408#diff-21ffe1f9a665c0f70cc9888d2da937a2a77fadfdacc23d0bd59506595e668c5fR267) and [InetAddress.java](https://github.com/openjdk/jdk/pull/13285/files/a856cd36eba1efacbc0f95918ac6aa2712cab870#diff-55d4acc64110543896bf6c289d6247cded986edfe9ccce3a9bca88a197711d58R204)

Reply:
In the [InetAddress.java](https://github.com/openjdk/jdk/pull/13285/files/a856cd36eba1efacbc0f95918ac6aa2712cab870#diff-55d4acc64110543896bf6c289d6247cded986edfe9ccce3a9bca88a197711d58R204)
```
 * <dt><b>networkaddress.cache.ttl</b></dt>
 * <dd>Indicates the caching policy for successful name lookups from
 * the name service. The value is specified as an integer to indicate
 * the number of seconds to cache the successful lookup. The default
 * setting is to cache for an implementation specific period of time.
```
In the [java.security](https://github.com/openjdk/jdk/pull/13285/files/ce595f74f511553274c9da254d53b368abe44408#diff-)
```
# default value is forever (FOREVER). For security reasons, this
# caching is made forever when a security manager is set. When a security
# manager is not set, the default behavior in this implementation
# is to cache for 30 seconds.
```

Reply:
Such specification allows changing the default property in JDK bundle w/o changing the doc.